src/acp/control.ts

```diff
@@ -167,34 +172,62 @@ export function getAccessControlAll(
  * function is still experimental and subject to change, even in a non-major release.
  * ```
  *
- * Insert an [[AccessControl]] into an [[AccessControlResource]], replacing previous instances of that Access Control.
+ * Insert an [[AccessControl]] into the [[AccessControlResource]] attached to a Resource, replacing
+ * previous instances of that Access Control.
  *
- * @param accessControlResource The Access Control Resource to insert an Access Control into.
- * @param accessControl The Access Control to insert into the given Access Control Resource.
- * @returns A new Access Control Resource equal to the given Access Control Resource, but with the given Access Control.
+ * @param withAccessControlResource A Resource with the Access Control Resource into which to insert an Access Control attached.
```

Comment:
Not sure what the final word 'attached' is referring to here - i.e. just remove that final word, or:
```
 * @param withAccessControlResource A Resource with the Access Control Resource into which to insert the specified Access Control.
```

Reply:
All references to "attached" have been removed: 93404f52fe8ab0e31bef93bf206b19c2214de6ba